__tests__/__system__/cli/submit/data-set/cli.submit.data-set.test.ts

```diff
@@ -61,6 +61,47 @@ describe("submit job from data set command", () => {
         expect(result.stdout.toString()).toContain("jobname");
     });
 
+    it("should be able to submit a job from a local file with wait option and get rc successfully", async () => {
+
+        // download the appropriate JCL content from the data set
+        const sleepDataSet = testEnvironment.systemTestProperties.jobs.sleepMember;
+        const jclFilePath = testEnvironment.workingDir + "/iefbr14.txt";
+        const wait = "3,5";
+        const result = runCliScript(__dirname + "/__scripts__/command/command_submit_data_set_wait.sh", testEnvironment, [sleepDataSet,wait]);
+        expect(result.stderr.toString()).toEqual("");
+        expect(result.status).toEqual(0);
+        expect(result.output.toString()).toContain("Waiting for job completion.");
+        expect(result.output.toString()).toContain("rc:");
+    });
+
+    it("should be able to submit a job from a local file with invalid wait value", async () => {
+
+        // download the appropriate JCL content from the data set
+        const sleepDataSet = testEnvironment.systemTestProperties.jobs.sleepMember;
+        const jclFilePath = testEnvironment.workingDir + "/iefbr14.txt";
+        const wait = "3,";
+        const result = runCliScript(__dirname + "/__scripts__/command/command_submit_data_set_wait.sh", testEnvironment, [sleepDataSet,wait]);
+        expect(result.stderr.toString()).toEqual("");
+        expect(result.status).toEqual(0);
+        expect(result.output.toString()).toContain("is invalid.");
+        expect(result.output.toString()).toContain("The default value");
+        expect(result.output.toString()).toContain("rc:");
+    });
+
+    it("should be able to submit a job from a local file but not finished within specified wait option", async () => {
+
+        // download the appropriate JCL content from the data set
+        const sleepDataSet = testEnvironment.systemTestProperties.jobs.sleepMember;
+        const jclFilePath = testEnvironment.workingDir + "/iefbr14.txt";
+        const wait = "1,2";
```

Comment:
Isn't it possible for an iefbr14 job to finish withing 2 seconds?

Reply:
Actually, the jclFilePath is not used and the submitted job is sleepDataSet. This job will sleep for 20s. This test case is for testing the job isn't  finished within specified wait option. Thank you zFernand0! I will delete the unused variables to avoid the confusion.